fluent-react/src/context.js

```diff
@@ -0,0 +1,4 @@
+import { createContext } from "react";
+import ReactLocalization from "./localization";
+
+export default createContext(new ReactLocalization([]));
```

Comment:
Absolutely, thanks for catching that! I fixed it and tried writing a test for it but then stopped because it was non-obvious how to best do that. Also I remembered that TypeScript is seriously considered which would make such a test obsolete (yay 🎉).